### DISCUSSION TITLE
[@exp] Rename useFirestoreEmulator?

### DISCUSSION THREAD

**Discussion by benmvp** [firebase/firebase-js-sdk#4735] [General]
Hi there,

The answer is probably "no," but I figured I might as well bring it up to see if others have run into this as well. So [`useFirestoreEmulator`](https://modularfirebase.web.app/reference/firestore_.usefirestoreemulator) is the equivalent of `db.useEmulator` in `@exp`. 

The problem is that for React applications `useFirestoreEmulator` looks like [React Hooks](https://reactjs.org/docs/hooks-intro.html). The code itself is fine, but it causes a false positive with the [`eslint-plugin-react-hooks`](https://www.npmjs.com/package/eslint-plugin-react-hooks). Because Hooks begin with `use*` and this is not being used within a React component it complains. 

I've disabled the warning, but just thought I'd see if there'd be any interest in renaming it to maybe `withFirestoreEmulator`? 😄 

Thanks!

**Comment by akauppi** [comment]
I still don't get why my source needs to state that it's going to be run under emulation. It's config, not code.
